task1/task1.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 1</title>
+</head>
+
+<body>
+    <table border=1 cellspacing=0 cellpadding=5px>
+        <thead>
+            <tr height=100px>
+                <th colspan="3">
+                    <font size="6">Markup of “Basic HTML” task</font>
```

Comment:
`<h*>`?

Reply:
<h*> tags have extra margin added by default in browsers so the size of header and aligning would change, and we cannot use CSS to correct that